readme.txt

```diff
@@ -39,7 +39,7 @@ Our global support team is available to answer questions you may have about WooC
 = Requirements =
 
 * WordPress 5.7 or newer.
-* WooCommerce 5.9 or newer.
+* WooCommerce 6.0 or newer.
```

Comment:
@asumaran 

I've just bumped the minimum version here as per this thread p1644907045226379/1644898556.442049-slack-C01BPL3ALGP